### PR TITLE
 Change the tab color to pink

### PR DESCRIPTION
**Pull request by Bekjo3**
Before : 
image

After: 
image

### CODE REVIEW

**Comment by Viannen10** [comment]
Please change your pull request title to something more descriptive

**Review by Viannen10 — COMMENTED** [review]
The color yellow is a bad color choice in this situation because the title is not viewable

File: styles.css [hunk 1]
```diff
@@ -21,7 +21,7 @@ body {
 }
 
 .tablink.selected {
-    background-color: darkred; /* Change to the color of your choice */
+    background-color: yellow; /* Change to the color of your choice */
```
Comment (Viannen10) [inline styles.css]: Yellow is not a good choice with a white font color (has bad contrast and inaccessible)

File: styles.css [hunk 2]
```diff
@@ -21,7 +21,7 @@ body {
...
     color: #fff; /* Change to the color that provides good contrast with the background */
```
Comment (Viannen10) [inline styles.css:25]: Alternatively, if you would like to keep the yellow background color please choose a better title color

**Comment by Bekjo3** [comment]
> The color yellow is a bad color choice in this situa

Which color would you prefer?

**Comment by Viannen10** [comment]
> > The color yellow is a bad color choice in this situa
> 
> Which color would you prefer?
#FF8B94
Screen Shot 2023-07-29 at 11 38 08 PM

**Comment by Bekjo3** [comment]
> > > The color yellow is a bad color choice in this situa
> > 
> > 
> > Which color would you prefer?
> > #FF8B94
> 
> Screen Shot 2023-07-29 at 11 38 08 PM

Sounds good

**Review by Viannen10 — APPROVED** [review]
Thank you for making the changes! LGTM

**Pull request merged**